platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/SupportMapFragment.java

```diff
@@ -28,7 +28,9 @@
  * </p>
  *
  * @see #getMapAsync(OnMapReadyCallback)
+ * @deprecated Starting with Android Pie, Android SDK fragments are deprecated. Use {@MapFragment} instead.
```

Comment:
This note is a bit confusing because we are deprecating `SupportMapFragment`. It'd worth to mention here that the `MapFragment` extends from `android.support.v4.app.Fragment` now.